ui/pages/confirmation/components/confirmation-network-switch/confirmation-network-switch.js

```diff
@@ -20,82 +18,54 @@ import {
 } from '../../../../../shared/constants/network';
 import { getProviderConfig } from '../../../../ducks/metamask/metamask';
 
-export default function ConfirmationNetworkSwitch({ newNetwork }) {
-  const { chainId, nickname, type } = useSelector(getProviderConfig);
-
+const NetworkIcon = ({ chainId, nickname }) => {
```

Comment:
We have a convention that files shouldn't contain multiple components I would suggest moving it to it's own file and creating a new storybook file or moving the code in this component back into the main component. There is some repetition but I think it would be fine